### ISSUE TITLE
Celery instead of AJAX admin

### ISSUE THREAD

**Issue by janvavra** [ECON3/dAuction2_interactive3#37]

**Comment by janvavra** [comment]
Partialy done, needs to ask Dmitrij about easiest way how to deamonize it.

**Comment by slvstr1** [comment]
I now cant run server. It gives: "ImportError: cannot import name Celery". And I have installed it.

**Comment by janvavra** [comment]
Strange, works for me. So comment out all parts where celery is present (celery.py file in dAuction2 and tasks.py in MASTER), then you should not need celery for anything.

**Comment by janvavra** [comment]
Destroyed and recreated vagrant box and it works for me, strange.

**Comment by slvstr1** [comment]
Stackflow says it is a confusion of the celery.py and the app. I renamed celery.py to celery_local.py and it works now.

**Comment by janvavra** [comment]
Good to hear. Its strange that is working fine on my Vagrant.

**Comment by slvstr1** [comment]
Yes, strange. I did, of course, destroy and recreate the vagrant box.

**Comment by janvavra** [comment]
Not needed now, wait for Dmitrij advice how to implement correctly.

**Comment by slvstr1** [comment]
We can assign this issue secondary importance. After all it works with ajax.
